src/arrayMethodSort.js

```diff
@@ -4,9 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
+  [].__proto__.sort2
+  = function(customCompareFunction = (a, b) => `${a}` > `${b}`) {
+      let compareFunction = customCompareFunction;
+
+      if (typeof compareFunction !== 'function') {
```

Comment:
The check for the type of compareFunction is unnecessary. The default value of the compareFunction ensures that it will always be a function.